templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -119,6 +119,14 @@ export class {{ service.name }}Client {
     const servicePath = opts?.servicePath || opts?.apiEndpoint || staticMembers.{{ id.get("servicePath") }};
     const port = opts?.port || staticMembers.{{ id.get("port") }};
     const clientConfig = opts?.clientConfig ?? {};
+    {% if api.rest -%}
+    // Implicitely set 'rest' value for the apis use rest as transport (eg. googleapis-discovery apis).
+    if (!opts) {
```

Comment:
let's do `if (!(opts && 'fallback' in opts)) - so if the user actually passed `fallback` to the constructor we used whatever the user passed